libdnf/module/module_goal_private.cpp

```diff
@@ -239,4 +239,23 @@ std::vector<std::vector<std::tuple<ProblemRules, Id, Id, Id, std::string>>> Modu
 }
 
 
+// Report packages that have a conflict
+libdnf::solv::IdQueue ModuleGoalPrivate::list_conflicting() {
```

Comment:
Do we need only packages with conflict. What about packages with broken dependencies?

Reply:
I am sorry, the code is same like in dnf4 therefore it is ok. I guess that packages with broken dependencies cannot negatively effect solver decision when jobs are optional.